CHANGES.rst

```diff
@@ -314,7 +304,7 @@ Miscellaneous internal changes
 
 
 
-- Restore :py:class:`~aiohttp.resolver.AsyncResolver` to be the default resolver. -- by :user:`bdraco`.
+- Restored :py:class:`~aiohttp.resolver.AsyncResolver` to be the default resolver. -- by :user:`bdraco`.
```

Comment:
That period in the middle could've been avoided.

Reply:
Do you think it’s worth adjusting before 3.10.1, or should we leave it as is?

Reply:
That's up to you. I know that it takes a lot of energy to make the change log nice. There's more things that could be fixed, like unlinked/unhighlited function mentions and perhaps other inconsistencies. I'd fix all of them if we were to work on improving it more systemically.